doc/source/tune/tutorials/overview.rst

```diff
@@ -5,7 +5,7 @@ User Guides
 ===========
 
 .. toctree::
-    :hidden:
+    :maxdepth: 1
```

Comment:
what do you think about about increasing this so that it is easier for readers to skim through the content?
```suggestion
    :maxdepth: 2
```